libs/stats/odc/stats/_cli_common.py

```diff
@@ -92,12 +92,25 @@ def click_yaml_cfg(*args, **kw):
     """
     def _parse(ctx, param, value):
         if value is not None:
-            from odc.io.text import parse_yaml_file_or_inline
-
-            try:
-                return parse_yaml_file_or_inline(value)
-            except Exception as e:
-                raise click.ClickException(str(e)) from None
+            import urllib
+            from urllib.parse import urlparse
+            r = urlparse(value)
```

Comment:
If you're going to use `.load_documents()`, it handles checking whether it's a local file or a remote file, so this shouldn't be necessary.